app/javascript/controllers/contributors_controller.js

```diff
@@ -2,7 +2,7 @@ import { Controller } from '@hotwired/stimulus'
 
 export default class extends Controller {
   static targets = [
-    'contributorTypePerson', 'contributorTypeOrganization', 'contributorTypePersonLabel',
+    'contributorTypePerson', 'contributorTypeOrganization',
```

Comment:
The toggle-y parts have been moved to the toggle controller.

Reply:
Nice!